contracts/BevorProtocol.sol

```diff
@@ -279,18 +279,15 @@ contract BevorProtocol is Ownable, ReentrancyGuard {
         // payout whatever remains from the vested funds to the auditor
         // as the difference between audit price and total withdrawn funds by auditors.
         uint256 totalWithdrawn = 0;
+        uint256 totalAmount = 0;
         for (uint256 i = 0; i < targetSchedules.length; i++) {
           VestingSchedule storage schedule = vestingSchedules[targetSchedules[i]];
-          uint256 vestedAmount = _computeReleasableAmount(schedule);
```

Comment:
I disagree, in these extreme cases where the money needs to be reverted all money possible should be returned. Won't make a difference if the auditor is diligent about withdrawls but even a cent extra we get back to wrongfully audited parties makes this distinction worth it.

Reply:
I guess the real edge case is: 10 month vesting duration, on month 9 it was determined an audit was invalidated. Auditor didn't manually withdraw at any point. They'll receive 0$, instead of the 90% that they theoretically could've had. I'm fine with either, because at the end of the day an audit was "invalid". Let's just keep what you have for now. It'll be at the auditor's discretion to maximize their payout. Meh.